### PR TITLE
add phantom for 4DBTi system

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

File: mne/viz/_3d.py [hunk 1]
```diff
@@ -1050,7 +1050,7 @@ def plot_alignment(info, trans=None, subject=None, subjects_dir=None,
 
     # initialize figure
     if fig is None:
-        fig = mlab.figure(bgcolor=(0.0, 0.0, 0.0), size=(800, 800))
+        fig = mlab.figure(bgcolor=(0.5, 0.5, 0.5), size=(800, 800))
```
Comment (agramfort) [inline mne/viz/_3d.py]: @larsoner @christianbrodbeck @dengemann I changed the background color for the 3d plot alignment. Looks like this now. I prefer the contrast now as we can better see the helmet.

screen shot 2018-02-01 at 10 09 33

**Comment by agramfort** [comment]
@massich have a look at all I had to do...

**Comment by agramfort** [comment]
here is the rendered example:

**Comment by agramfort** [comment]
here is the rendered example:

https://6874-1301584-gh.circle-artifacts.com/0/tmp/circle-artifacts.vywqNIB/html/auto_tutorials/plot_phantom_4DBTi.html

**Comment by larsoner** [comment]
PEP8

https://travis-ci.org/mne-tools/mne-python/jobs/336009474#L2990

**Comment by massich** [comment]
Travis [failed](https://travis-ci.org/mne-tools/mne-python/jobs/336009477#L2372) but I think it was a random failure. I corrected the pep8 error, lets see what travis has to say this time

**Comment by agramfort** [comment]
thanks everyone

**Pull request merged**